src/app/shared/search/search-filters/search-filter/search-range-filter/search-range-filter.component.scss

```diff
@@ -26,7 +26,7 @@
     }
 
   .noUi-connect {
-      background: var(--ds-slider-color);
+      background: darken($green, 20%);
```

Comment:
Same comment here regarding sass variables. Since it's a global class, I'd move it out of this component style

Reply:
I restored the `--ds-slider-color` here as you requested. I also found out that I used `$dark` for handle slider, so I turned that into a css variable too (`--ds-slider-handle-color`).